src/wallet/coincontrol.h

```diff
@@ -30,6 +30,8 @@ class CCoinControl
     boost::optional<CFeeRate> m_feerate;
     //! Override the default confirmation target if set
     boost::optional<unsigned int> m_confirm_target;
+    //! Avoid partial use of funds sent to a given address: -1 means default
+    short m_avoid_partial_spends;
```

Comment:
what about `std::optional<bool>` ?

Reply:
(since C++17) 😛

Reply:
Ah I forgot we were using boost for that... I would still prefer boost than a short though.

Reply:
Changed to using a `boost::optional`.